modules/electromagnetics/include/auxkernels/AuxComplexHeating.h

```diff
@@ -0,0 +1,28 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "AuxKernel.h"
+
+class AuxComplexHeating : public AuxKernel
+{
+public:
+  static InputParameters validParams();
+
+  AuxComplexHeating(const InputParameters & parameters);
+
+protected:
+  virtual Real computeValue() override;
```

Comment:
but no need for that one, because it inherits it from the base implenentation

Reply:
Ok, thanks for the clarification. I have quite a few files that need docstrings, so I will address all those files and comment in this thread once all the files are finished.

Reply:
I added the missing docstring for the new classes and attributes.